install_themes/action.yml

```diff
@@ -54,3 +54,55 @@ runs:
             ${{ inputs.container }} \
             composer update ${{ inputs.update_options }}
         fi
+
+    - name: Generate deprecation warning to summary page
+      shell: bash
+      run: |
+        echo "Deprecation Warning:" >> $GITHUB_STEP_SUMMARY
+          echo "---------------------" >> $GITHUB_STEP_SUMMARY
+          echo -e "\033[0;31m### This actions are deprecated and may not be supported ${{ inputs.install_theme}} \033[0m" >> $GITHUB_STEP_SUMMARY
```

Comment:
There is no input variable named "install_theme". So, instead, you can either just write "install_theme" or use ${{github.action}} if you want thename of the action to be placed here dynamically.
See [the context documentation](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) for more possible variables.